.flowconfig

```diff
@@ -100,7 +100,8 @@ esproposal.optional_chaining=enable
 esproposal.nullish_coalescing=enable
 
 # The main warning is "Unused suppression comment".
-include_warnings=true
+# Temporarily disabled to smooth the RN upgrade.
+# include_warnings=true
```

Comment:
I think you want this a few commits earlier in the branch, before "flagsReducer: Add a $FlowFixMe for something Flow v0.113 will catch."

Reply:
How about moving

"flagsReducer: Add a $FlowFixMe for something Flow v0.113 will catch."

later in the branch? I think this is the only one that's strayed away from the unused-fixmes-allowed window.

Reply:
Oh, hang on—actually I think there's a subtlety that I didn't keep track of. If the Flow upgrade proceeded without the changes in

"flagsReducer: Add a $FlowFixMe for something Flow v0.113 will catch."

, there would be an error at a different site than where that `$FlowFixMe` is:

```
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ src/chat/flagsReducer.js:48:10

Cannot return object literal because inexact object literal [1] is incompatible with exact FlagsState [2].

 [2] 33│ ): FlagsState => {
       :
     45│     });
     46│   });
     47│
 [1] 48│   return {
     49│     ...state,
     50│     ...newState,
     51│   };
     52│ };
     53│
     54│ const removeFlagForMessages = (state: FlagsState, messages: number[], flag: string): FlagsState => {
```

The way I suppress that error is to pretend that `{}` for `newState` is a `FlagsState`, which is a lie, and that lie gets caught even before v0.113.

Reply:
Mmm, I see.

Yeah, that strategy makes sense. In describing it, I'd just emphasize that you're casting these values, lying about their types. The change isn't primarily about adding a fixme (as it might be for covering an existing lie / type error that Flow is about to start spotting) as adding a new lie, with a fixme to cover it up.